var/spack/repos/builtin/packages/dcm2niix/package.py

```diff
@@ -11,8 +11,12 @@ class Dcm2niix(CMakePackage):
     """DICOM to NIfTI converter"""
 
     homepage = "https://github.com/rordenlab/dcm2niix"
-    url = "https://github.com/rordenlab/dcm2niix/archive/refs/tags/v1.0.20210317.tar.gz"
+    url = "https://github.com/rordenlab/dcm2niix/archive/refs/tags/v1.0.20220720.tar.gz"
+    list_url = "https://github.com/rordenlab/dcm2niix/tags"
```

Comment:
@tldahlgren did something change in `spack checksum` recently that broke support for GitHub URLs? This used to work automatically.

I would remove this change here and open an issue to track this, we should be able to fix this for all GitHub URLs simultaneously.

Reply:
I was not aware that this should work automatically. I have opened an issue and removed `list_url` here.
Thanks.